x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.tsx

```diff
@@ -151,20 +159,28 @@ export const AllCases = React.memo(() => {
     [isDisplayConfirmDeleteModal]
```

Comment:
The dependency seems not exactly match with what is using inside the function.

Reply:
you're right, code bloat for sure. ill get rid of it